dspace-api/src/main/java/org/dspace/content/DSpaceObject.java

```diff
@@ -36,7 +37,8 @@
 @Entity
 @Inheritance(strategy = InheritanceType.JOINED)
 @Table(name = "dspaceobject")
-public abstract class DSpaceObject implements Serializable, ReloadableEntity<java.util.UUID> {
+public abstract class DSpaceObject implements Serializable, ReloadableEntity<java.util.UUID>,
+    BrowsableDSpaceObject<UUID> {
```

Comment:
you are right the name is not appropriate... the reason for that is that this interface exists in dspace-cris since many years and was introduced to allow "browsing" of different object other than item. Over the time the purpose is evolved but we have never renamed it.

Now, we are introducing it in DSpace 7 to allow objects to be searchable in Discover, so a more appropriate name could be SearchableObject or DiscoverSupport or ... ?

Reply:
Renaming this to be `BrowseableObject` or `SearchableObject` is fine by me.  It shouldn't say `DSpaceObject` in the name though, as it's definitely not limited to `DSpaceObject`s

Reply:
done